docs/tutorials/12_deuteron_binding_energy.ipynb

```diff
@@ -0,0 +1,1286 @@
E8/vjjbLXVVhx44IEMGDCAK6+8km222SZvYtvee+/NMcccw4ABAwA49NBDOeaYYzjmmGPYbLPNssddcskl7Ljjjtx0002st956HHTQQWy22WbceeedTJo0ib/+9a9lx3rqqady4okn0tjYyP7778+2226bDXRuvPFGxo8fz9VXX82AAQPYf//9+cIXvsBjjz3GQQcdxM9//vOyt3vVVVcxefJkVq5cyb777suWW27JM888w5QpU7jllluKjl+6dCmTJk3iu9/9Lk899RQTJkzgkEMOYeONN+aJJ57gpz/9aZeeS9+xxx6LmXHssceWHXvYXX311QAcffTRFQO90047jUWLFnHllVeWrGXPtXLlSqBycOvve/755zs75IrefPNNzjnnHE488USmTp3K9ddfnx1PZ8c5aNCg7ByBwnH6kz7PP/98Vq1ald3unOO8885j1apVHHjggayzzjodGrd/kl1q/sH777/Pf/7zH6LRKDvuuCOvv/465513Ht/97neZNm0a99xzT6cTWW+//TZ//vOfAS/Q7Yw33ngDgPXXX79T1+uIW2+9ldNOO42TTjqJc889l8cee6yq27n77rt56KGH6N+/P3vuuWfePn+S8HbbbVfyuv72N998s+x7p+ZUm/qv10s9lejMeecTN+asO9wjCz4Keigddsm9C9yYs+5w1zz1TtBD6ZK/PvpfN+asO9ykix/Mbrvw7lfdmLPucJtOuzPAkbVPJTrtq1Si85///MdFo9GSNfhnnXWWA9wuu+xStO+Pf/yjA9ymm26aV1+ae3+lSnT8EoyRI0cWlQk8/vjjbvDgwSVrb8l8TT1kyBD3zDPPFN3uiy++6BoaGtzAgQPdXXfdVfQYR48e7QD34IMP5u3zx9rQ0ODuvvvuvH3nnXeeA9xmm21WdH8HHXSQA9yOO+5YVEfb0tJSNIZqn8tjjjmmqrrvzuqpEp23337bmZkD3Isvvlj2uNtuu80B7qSTTspuq1Sic9999znADR8+vOTtJRIJ19DQkK19r6SzJTqlLmuttZa7+eabi67z17/+1QFu4sSJJW9z4cKF2duYOnVq3r7Fixdn6/eHDx/u9t9/f3fooYe6zTbbzMXjcXf00Ue3W37ke/jhh7Pv81KlHrNnz3aAW2edddz//d//uVgsVvQYd9ppJ/fhhx+WvY+rr77aHXPMMe7rX/+622WXXVwsFnORSKTkPIhKXnrpJReLxZyZuXnz5rV7fGdLdEpddt55Z7dwYeVS2zPPPNMdc8wx7vDDD3df/OIXHeAGDRrkZs6cWXSsPwdk1qxZZW9v8ODBDnDz58+veL/+GMNeohN4wNypwcJY4DTgOuA1IJ15og/rwHW/DjwGfAqsBOYA3wMinRlDPQX48/+3zI056w53z3/eD3ooHTb9ntfcmLPucDOeeDvooXTJpQ++4cacdYfb+cIHstt+fecrbsxZd7iNzr4jwJG1ryN/vCZNmlR0+dOf/uScc+6zzz4ruf9vf/ubc877T7bUfr+WeOHChSX333bbbc4551577bWS+++77z7nnHNz584t2tfdSgX4S5YscXfddZfbYostnJm5yy67LO86n3zyiWtqanIDBw4s+5/6fvvt54DsYy28v1IBvh+wFAbAvunTp2drsHP5/8n9+te/Lnm9I444wgFFj8Pn1zofcsghJcf64x//uOg6a9ascUOGDHGAe/fdd7Pb586dm/3P/aOP2k9IdOW5PPvss93YsWPd2Wef3e79dEVPBfg///nPKwa4znnvxfXXX9+NHj06L2CtFOAvX77c9e/f3wHulltuKdr/l7/8Jfue2WuvvSqOsaOP/Z577nHnnHOOe/bZZ93HH3/sli1b5p566il38MEHO8BFIhF3zz35c5befPNNZ2bOzEoGoNOmTcuO88QTTyzav3LlyuxJXu5l7Nix7q9//WvF8eb61re+VfFk54YbbnBANrA/6qij3CuvvOKWL1/uHnzwQTdu3LjsCWo5xx9/fN4YY7GY+/Wvf100WbeSFStWuM9//vMOcN/+9rc7dJ2OBvjnnHOOu+qqq9zrr7/uVq1a5d599113ww03ZOcMbbHFFiUnC/s23XTTvMc3fPjwku8955zbfPPNHZD9O1/KyJEjy86xyFUrAX6tleicDPwO+AZesG8Vj84wsz8B/wAm4gX59wFbAJcCt5hZrT0PvaKpwa/Br50SHb+kJe1qu4wlUaoGP/Nv/8Rcat9uu+2WrWUfNmwY++67L++++y533303J598ct6xDz30EC0tLUyaNKlsCYDf7vCpp57q0P1//PHHPPvsswwePDhbe9zZ2zzkkEOKtqXTae655x7MjMMOO6yq2y3Va72hoYFNNtkEgEWLFmW333PPPYBXdjBixIiSt5erK8/lBRdcwGuvvcYFF1zQ7v2ETTqdzpaFfPvb3y573Kmnnsr777/PX/7yl5L1yqUMGjSIH//4x4BXcjJjxgwWL17MBx98wGWXXcaPfvQj4vE4QF57xq6YPHky5513Httttx1rr702Q4YMYYcddmDmzJmcfvrppNPp7Jh8m266KUcffTTOOQ466CBuvfVWli5dysKFCznvvPOYPn162XG+9tprTJgwgdmzZ3Pttdfy/vvvs2zZMh544AEGDBjACSecUPF59S1fvjxbZlbu+HRmPlkymeTLX/4y119/PePGjWPQoEHstttu3HvvvTQ1NfHoo4/y0EMPlbyNK6+8Euccq1at4uWXX+aHP/whv/zlL9lhhx3yPj/lJBIJDj/8cP7zn/8wfvx4/vCHP7R7nc4477zz+Pa3v83mm29OU1MTG264IUceeSRz585lk0024fXXX+fyyy8ve/0333wT5xxLly7lySef5Mtf/nK27K7cmgv1pNb64P8HmI6XfX8euAqYVOkKZnYocArwAbCLc+6NzPZ1gYeAg4FTgd/33LBrU7947QX4fmBf6xNRS9fgt00sSDuIduj0Npwefvjhsvv69+9fcf/w4c
```

Comment:
You should either add a title to this plot, or an explanation/description prior to the cell